src/Apps/Order/Components/__tests__/StickyFooter.test.tsx

```diff
@@ -66,4 +69,88 @@ describe("Sticky footer", () => {
       "Need help? Read our FAQ or ask a question."
     )
   })
+
+  describe("Analytics", () => {
+    describe("on a make offer page", () => {
+      it("tracks click on 'Read our FAQ'", () => {
+        const { Component, dispatch } = mockTracking(StickyFooter)
```

Comment:
Yep! I think it didn't work for us before because we were missing this: jest.unmock("react-tracking")

Reply:
ahhh that makes sense. glad this worked out and sorry for the confusion around testing this, not the first :)

Reply:
Yah 👍

Side-note, it would be cool if we could submit a jest PR that adds a prop to mocks so it can be recognised as such, instead of having to rely on knowing exactly where the mock is defined and using that instance to do an identity comparison. /cc @orta